src/design/optForce/analizeOptForceSol.m

```diff
@@ -0,0 +1,131 @@
+function [maxGrowthRate, minTarget, maxTarget] = analizeOptForceSol(model, targetRxn, solution, relax, tol)
```

Comment:
Just a hint:
When renaming files:
use git mv originalFileName newFileName
That way, the files get tracked and its much easier to check earlier changes.